### ISSUE TITLE
Using trackpad horizontal scroll not working 

### ISSUE THREAD

**Issue by asp3** [asmyshlyaev177/react-horizontal-scrolling-menu#107]
**Describe the bug**
On mac devices, there is a native way to do horizontal scrolling (swipe the trackpad to the left or right with multiple fingers). However, when I do this, the menu bounces back, and sometimes, the scrolls properly (see video attached)

**To Reproduce**
```
            <ScrollMenu
                data={classItems}
                arrowLeft={ArrowLeft}
                arrowRight={ArrowRight}
                hideArrows={true}
                hideSingleArrow={true}
                alignCenter={false}
            />
```

**Screenshots**
[Video of Issue](https://streamable.com/stsnl7)

**Desktop (please complete the following information):**
 - OS: macOS v. 10.15.2
 - Browser Chrome v. 81.0.4044.138
 - **react-horizontal-scrolling-menu v. 0.7.7**

**Comment by asp3** [comment]
any progress on this?

**Comment by jitbasemartin** [comment]
Same issue here on Macbook pro trackpad and Firefox. I use `Shift + vertical scroll` but it's not intuitive and very sensible to use

**Comment by asmyshlyaev177** [comment]
Published version `2.0.0` there can do this.
